e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -203,13 +204,27 @@ describe("scenarios > dashboard > dashboard cards > click behavior", () => {
 
       saveDashboard();
 
+      cy.intercept(
+        "GET",
+        "/api/collection/root",
+        cy.spy().as("rootCollection"),
+      );
+      cy.intercept("GET", "/api/collection", cy.spy().as("collections"));
+
       clickLineChartPoint();
       cy.get("@targetDashboardId").then(targetDashboardId => {
         cy.location().should(({ pathname, search }) => {
           expect(pathname).to.equal(`/dashboard/${targetDashboardId}`);
           expect(search).to.equal("");
         });
       });
+
+      cy.log("Should navigate to question using router (metabase#33379)");
```

Comment:
Piggyback assertions that test the navigation have been done using router.

If you check out `master` and run this test. The test should fail.